framework/origin_trials_client.py

```diff
@@ -163,20 +164,29 @@ def create_origin_trial(ot_stage: Stage) -> str | None:
       'type': ('DEPRECATION'
                 if ot_stage.ot_is_deprecation_trial else 'ORIGIN_TRIAL'),
     },
-    'internal_registration_config': {'approval_type': 'NONE'},
+    'registration_config': {'approval_type': 'NONE'},
+    'trial_contacts': []
   }
+
+  # Get a list of all OT @google.com contacts (ot_owner_email must be a google
+  # contact).
+  unique_contacts = [ot_stage.ot_owner_email]
+  unique_contacts.extend(ot_stage.ot_emails)
+  unique_contacts = [email for email in set(unique_contacts)
+                     if email.endswith('@google.com')]
```

Comment:
Group contacts need to be filtered as well, since they cannot be added to Ganpati or Google group APIs. But that is much harder to do here. We need to do it on the OT side